recipe/meta.yaml

```diff
@@ -13,19 +13,26 @@ source:
 
 build:
   number: 0
-  skip: true  # [win]
   detect_binary_files_with_prefix: True
 
 requirements:
   build:
-    - autoconf
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}        # [not win]
+    - {{ compiler('m2w64_c') }}  # [win]
+    - {{ compiler('cxx') }}        # [not win]
+    - {{ compiler('m2w64_cxx') }}  # [win]
+    - m2-autoconf  # [win]
+    - m2-automake1.15  # [win]
+    - automake  # [not win]
+    - autoconf  # [not win]
```

Comment:
please, let this as before, and you also have to bump the build number

Reply:
thanks for stepping in @marcelotrevisani!

what do you mean by "as before"? what should I remove/add back?

ok for the build number, I forgot that, thanks!

Reply:
let the recipe as before (without your modifications in the ``meta.yaml``) and add
```
    - m2-base  # [win]
    - m2-make  # [win]
```
in build requirements

Reply:
ok, trying now, but won't there be many missing host dependencies on windows (bison, flex, gperf)...?  I'm shooting in the dark here...

Reply:
I must have completely missed the point of `{{ compiler('m2w64_cxx') }}`...

Reply:
FYI: instructions for win builds here: http://iverilog.wikia.com/wiki/Installation_using_MSYS2

Reply:
@marcelotrevisani any more hints?